frontend/src/copyButton/CopyButton.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { theme } from '../styles/theme';
+import type { CopyButtonVariant } from './variant';
+
+const color = {
+  filled: css`
+    background-color: ${theme.colors.gray06};
+    color: ${theme.colors.white};
+    &:active{
+      background-color: ${theme.colors.gray05};
+    }
+  `,
+  outlined: css`
+    background-color: ${theme.colors.white};
+    border: 1px solid ${theme.colors.gray06};
+    color: ${theme.colors.gray06};
+    &:active{
```

Comment:
```suggestion
    &:active {
```

공백 추가하면 좋겠네요!
이 외에도 스타일 코드에 prettier가 제대로 적용되지 않은 것 같은데, 추후에 모아서 같이 리팩토링해봐요☺️